tests/unit_test/data_processor/data_processor_test.py

```
@@ -7118,6 +7118,159 @@ def test_edit_custom_responses(self):
+                "type": "catalog",
+                "data": {
+                    "page_name": "store_home",
+                    "identifierType": "value",
+                    "identifierValue": "acme",
+                    "label": label_value
+                }
+            }
+            utterance_name = f"utter_catalog_empty_label_{label_value.strip() or 'empty'}_test"
+            assert processor.add_custom_response(
+                catalog_payload, utterance_name, "tests", "testUser"
+            ), f"Save should succeed for label={repr(label_value)}"
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)